test/integration/lib/config/foxbox.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+
+	nupnp_server : {
```

Comment:
ah, it's just my IDE interprets tabs as 2 spaces.  I have to replace the tab to 2 spaces